common-library/pom.xml

```diff
@@ -37,6 +37,11 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-tx</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.opencsv</groupId>
+            <artifactId>opencsv</artifactId>
+            <version>5.7.1</version>
```

Comment:
Can you move the version to parent pom.xml? 
<opencsv.version>5.7.1</opencsv.version>